recipes/jasper/all/test_package/CMakeLists.txt

```diff
@@ -1,16 +1,7 @@
 cmake_minimum_required(VERSION 3.1)
 project(test_package C)
 
-include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
-conan_basic_setup(TARGETS)
-
-find_package(Jasper REQUIRED)
-
-message("JASPER_FOUND: ${JASPER_FOUND}")
-message("JASPER_INCLUDE_DIR: ${JASPER_INCLUDE_DIR}")
-message("JASPER_LIBRARIES: ${JASPER_LIBRARIES}")
-message("JASPER_VERSION_STRING: ${JASPER_VERSION_STRING}")
```

Comment:
could you test that these variables are defined?

Reply:
Done.